config/application.rb

```diff
@@ -40,12 +40,6 @@ class Application < Rails::Application
 
     config.middleware.use Rack::Deflater
 
-    # Replace with a lambda or method name defined in ApplicationController
-    # to implement access control for the Flipflop dashboard.
-    config.flipflop.dashboard_access_filter = -> {
-      head :forbidden unless current_user.has_any_role?(:super_admin)
-    }
-
```

Comment:
This made it so that only super admins had access to toggle feature flags. Is there something similar we can add for sail?

Reply:
I will add the support for defining authorization lambdas in Sail

Reply:
Awesome